packages/babel-preset-jest/index.js

```diff
@@ -8,7 +8,6 @@
 module.exports = () => ({
   plugins: [
     require.resolve('babel-plugin-jest-hoist'),
-    require.resolve('@babel/plugin-syntax-object-rest-spread'),
-    require.resolve('@babel/plugin-syntax-bigint'),
+    require.resolve('babel-preset-current-node-syntax'),
```

Comment:
we could add the 3 the new preset adds manually, then replace all the plugins we add with this new preset in 26. We already add bigint and object spread anyways, so there's precedence

Reply:
opened #9775 with that